elements/campaign-display/fields/campaign-request-field.js

```diff
@@ -19,7 +19,7 @@ const CampaignRequestField = {
 
     fetchCampaignSuccess(state, response, store) {
       state.requestInFlight = false;
-      store.actions.handleFetchComplete(response);
+      setImmediate(() => store.actions.handleFetchComplete(response));
```

Comment:
By which I mean the traditional sense of immediately, not the setImmediate nonsense way.